src/encoder/stateless/h264/predictor.rs

```diff
@@ -28,6 +28,9 @@ use crate::encoder::stateless::FrameMetadata;
 use crate::encoder::RateControl;
 use crate::encoder::Tunings;
 
+pub const MIN_QP: u8 = 1;
+pub const MAX_QP: u8 = 51;
```

Comment:
By itself this CL only triggers a few warnings and does nothing noticeable, let's merge it into the next one. Also should this be `pub(crate)`?

Reply:
Done :D